packages/components/pie-button/src/defs.ts

```diff
@@ -1,6 +1,8 @@
 export const sizes = ['xsmall', 'small-productive', 'small-expressive', 'medium', 'large'] as const;
 export const types = ['submit', 'button', 'reset', 'menu'] as const;
-export const variants = ['primary', 'secondary', 'outline', 'ghost', 'inverse', 'ghost-inverse'] as const;
+export const variants = [
```

Comment:
you just need to update the readme with the new prop values and this looks good to go!